cmsis-core/cmsis_nvic.h

```diff
@@ -34,6 +34,12 @@
 
 #include "cmsis.h"
 
+/* throw meaningful compiler error if yotta configuration is not provided */
+#if !defined(YOTTA_CFG_CMSIS_NVIC_USER_IRQ_OFFSET)      || !defined(YOTTA_CFG_CMSIS_NVIC_USER_IRQ_NUMBER) || \
```

Comment:
Isn't this a better place for those defines? https://github.com/ARMmbed/cmsis-core/blob/master/cmsis-core/core_generic.h